File: FLM_SubconLabelSystem/Library/Library.Database/DAL/MM_PRINT_ALIGN_INIT.cs
using System.Data;
using System.Data.SqlClient;

namespace Library.Database.DAL
{
    public class MM_PRINT_ALIGN_INIT : Library.SQLServer.Connection
    {
        public MM_PRINT_ALIGN_INIT() : base("PFR_Label_DB") { }

        internal DataTable Print_Align_init()
        {
            DataTable _obj_dt = new DataTable();

            base._cmd.CommandText = "SP_PRINT_ALIGN_INIT";
            base._cmd.CommandType = CommandType.StoredProcedure;
            base._cmd.CommandTimeout = 0;
            base._cmd.Parameters.Clear();

            _obj_dt.Load(base._cmd.ExecuteReader());
            return _obj_dt;
        }
    }
}

File: FLM_SubconLabelSystem/Library/Library.Database/DAL/MM_PRINT_ALIGN_INIT.vb
Imports System.Data.SqlClient
Imports System.Text
Namespace DAL

    Public Class MM_PRINT_ALIGN_INIT
        Inherits Library.SQLServer.Connection
        Public Sub New()
            MyBase.New("PFR_Label_DB")
        End Sub

        Friend Function Print_Align_init()

            Dim _obj_dt As New DataTable

            With MyBase._cmd
                .CommandText = "SP_PRINT_ALIGN_INIT"
                .CommandType = CommandType.StoredProcedure
                .CommandTimeout = 0
                .Parameters.Clear()

            End With

            _obj_dt.Load(MyBase._cmd.ExecuteReader)

            Return _obj_dt

        End Function


    End Class

End Namespace
